public/scripts/controllers/aboutController.js

```diff
@@ -11,7 +11,11 @@ var app = app || {};
   aboutController.initAbout = function(){
     $('.tab-content').hide();
     $('#about-me-page').fadeIn(350);
+    $('#about').fadeIn(350);
+
+    app.repos.requestRepos(app.repoView.index);
```

Comment:
I wanted to display my GitHub repositories on my About Me page for this assignment. Did it need to be somewhere else?